registry.json

```diff
@@ -1727,5 +1727,20 @@
       "telegram":"manizzler",
       "operatorName":"manizzler",
       "github":"manizzle"
+   },
+   {
+      "ticker":"MEME",
+      "poolId":"c9aa8e864d5cc292f0e93669a79b41d67f8ea62481a0d85b14bea386",
+      "social":{
+         "facebook":"",
+         "twitter":"https://twitter.com/meme_pool",
+         "website":"https://twoangrycamelsinacar.com/",
+         "reddit":"https://www.reddit.com/user/MEME-Pool",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"sporked",
```

Comment:
Can't find this username on Telegram.  We need a valid username for SPA voting.

Reply:
Whoops, typo. Fixed and merged in the latest changes.